### PR TITLE
Create home page according to mockups

### PR DESCRIPTION
**Pull request by Jasper-Nelligan**
Adds icons and buttons according to mockups

Screenshot (17)

### CODE REVIEW

File: app/src/components/largeButton.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default function LargeButton({text}) {
+  return (
+    <button className="bg-blue-500 hover:bg-blue-700 border border-blue-500 text-white font-bold py-3 px-6 rounded-lg text-4xl">{text}</button>
+  );
+}
```
Comment (Jasper-Nelligan) [inline app/src/components/largeButton.js:5]: Should I create a new class like this? Or should we just use the regular button class and allow us to specify text size and button size in the props?

**Comment by mas393** [comment]
Screenshot from 2024-02-12 09-03-41

Maybe this should be a separate issue, but the "I need printing/I can print" text doesn't scale well when changing the zoom level

**Comment by Jasper-Nelligan** [comment]
> Screenshot from 2024-02-12 09-03-41
> 
> Maybe this should be a separate issue, but the "I need printing/I can print" text doesn't scale well when changing the zoom level

I'll file a separate ticket for this since we're tight for time for milestone 1

**Comment by J-Raymer** [comment]
The responsiveness of this page will need to be addressed after this weeks deadline but i will go ahead and merge. I added your bug @Jasper-Nelligan to the milestone 2 project.

**Pull request merged**